docs/content/dev/setup/frontend.md

```diff
@@ -52,3 +52,10 @@ You can inspect the generated production-bundle files to look for optimization i
 
 1. run `yarn analyze`. This will overwrite any existing builds!
 2. Open the generated `.next/server/analyze/server.html` in your favourite browser
+
+## Enable Logging in Production
+
+Logger can be enabled in production by setting debugLogging in localstorage to true.
+```
+window.localStorage.setItem("debugLogging", 'true');
+```
```

Comment:
```suggestion
This can be done e.g. by executing this JavaScript command in the browser's console.

\```javascript
window.localStorage.setItem("debugLogging", 'true');
\```
```

(Without the `\`. GitHub was just weird.)